doc/api/next_api_changes/2018-09-18-DS-pylab.rst

```diff
@@ -0,0 +1,52 @@
+Removal of deprecated :mod:`matplotlib.pylab` code
+--------------------------------------------------
+
+Lots of code inside the :mod:`matplotlib.mlab` module which was deprecated
+in Matplotlib 2.2 has been removed. This means the following functions are
+no longer available in the `matplotlib.pylab` module:
+
+  - ``amap``
+  - ``base_repr``
+  - ``binary_repr``
+  - ``bivariate_normal``
+  - ``center_matrix``
+  - ``csv2rec`` (use `numpy.recarray.tofile` instead)
```

Comment:
Does `csv2rec` have a replacement? `tofile` seems like a replacement for `rec2csv` not `csv2rec`.

Reply:
I suggest using `pandas` instead.  If you really need csv -> record array the best course of action is probably to vendor the code to your project.